geminidr/gemini/lookups/timestamp_keywords.py

```diff
@@ -11,23 +11,23 @@
     "addObjectMaskToDQ": "ADOBJMSK",
     "addReferenceCatalog": "ADDRECAT",
     "addVAR": "ADDVAR",
+    "adjustWCSToReference": "CORRWCS",
+    "adjustSlitOffsetToReference": "CORRSLOFF",
```

Comment:
@chris-simpson - Is that ok for the timestamp keyword (`CORRSLOFF`) ?

Reply:
Oups, it seems I'm having trouble counting to eight :). Let's go for `CORRSLOF` then ?